tests/js/event_actions.spec.js

```diff
@@ -297,3 +297,32 @@ test('action parameter can use double-quotes', async () => {
         expect(payload.actionQueue[0].payload.params).toEqual(['double-quotes are ugly', true])
     })
 })
+/* How can I correctly test for this?
```

Comment:
maybe this is a good [starting point](https://stackoverflow.com/questions/52224447/jest-unit-test-for-a-debounce-function#answer-52226973)